### DISCUSSION TITLE
RFSTAR RF-DG-52PAS CC2652P

### DISCUSSION THREAD

**Discussion by chris-1243** [Koenkk/Z-Stack-firmware#492] [General]
@Koenkk I can not find this model in your list. I would like to update it but I have no idea which firmware I should use. I'm pretty sure is based on RF-Star module with TI CC2652P chip. That would mean I should such firmware "CC1352P2_CC2652P_launchpad_*.zip"

Any advice would be really appreciated

[RFSTAR RF-DG-52PAS CC2652P](https://www.rfstariot.com/rf-dg-52pas-cc2652p-usb-dongle_p87.html)

**Comment by dinhchinh82** [comment]
> CC1352P2_CC2652P_launchpad_*.zip

I have same hardware and it works well with CC1352P2_CC2652P_launchpad_*.zip.

**Reply by chris-1243** [reply]
Great. Many thanks for your feedback.
It will be used as test a for backup in any case my Sonoff Dongle-P fails

**Comment by chris-1243** [comment]
@dinhchinh82 Could you provide me which tool you used to flash it. I am trying with JelmerT/cc2538-bsl without any success. Any special actions maybe to run ? I am a bit stuck

EDIT: I had to press the boot button when plugging the dongle to enter in bsl mode... flash completed
